src/style.css

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 406px;
+  transition: 0.3s;
+  cursor: pointer;
+  box-sizing: border-box;
+}
+
+.product-card__logo {
+  height: 134px;
+  width: 160px;
+  margin-top: 16px;
+  margin-bottom: 40px;
+
+}
+
+.product-card__info {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  height: 54px;
+  margin-bottom: 16px;
+}
+
+.info_name {
+  display: contents;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  font-weight: 500;
+}
+
+.info-code {
+  margin-top: 4px;
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+}
```

Comment:
This class naming does not comply with the BEM methodology

Reply:
fixing...